### ISSUE TITLE
Master branch?

### ISSUE THREAD

**Issue by irrg** [nuxt/nuxt.js#3177]
I'm using Nuxt 1.4.x and I'm running into opencollective-cli related issues (documented in other issues on this repo) and wanted to try bumping opencollective-cli's version to 1.0.4. 

To my surprise I found that the `master` branch is kind of abandoned, and is at v1.2? `dev`'s at 2.0.0 which I didn't think was out yet. It feels a little sudden since I read this article this morning — https://medium.com/nuxt/nuxt-2-is-coming-oh-yeah-212c1a9e1a67 — and I feel like I didn't see any RCs for 2.0.

TL;DR shouldn't `master` contain the most current stable build?

**Comment by pi0** [comment]
@irrg You are right. We have to sync master with dev on each `latest` release. BTW you can also use tag branches (https://github.com/nuxt/nuxt.js/tree/v1.4.0).

**Comment by irrg** [comment]
Thanks, that makes sense. I like tags, but ![](https://i.imgur.com/SxprH.gif)

**Comment by irrg** [comment]
@pi0 Is there any chance that master will be shored up to be `tags/v1.4.0` equivalent soon? Also, I have a PR for it that could slide right into it that's related to a bunch of opencollective issues (see https://github.com/nuxt/nuxt.js/pull/3178 )

**Comment by irrg** [comment]
Hi gang,

Is there any way `master` could be updated to reflect the last 1.4.x release before 2.0?
